src/main/java/seedu/address/logic/commands/AddTransactionCommand.java

```diff
@@ -59,15 +62,18 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
         if (model.getIsViewTransactions()) {
+            logger.fine("CommandException caused by attempt to use addt command in transaction view.");
             throw new CommandException(String.format(Messages.MESSAGE_MUST_BE_PERSON_LIST, COMMAND_WORD));
         }
 
         List<Person> lastShownList = model.getFilteredPersonList();
         if (index.getZeroBased() >= lastShownList.size()) {
+            logger.fine("CommandException caused by invalid index.");
```

Comment:
maybe can update this message to `CommandException caused by invalid person index provided`